bindings/java/src/main/com/apple/foundationdb/tuple/Tuple.java

```diff
@@ -483,6 +483,15 @@ public List<Object> getItems() {
 		return new ArrayList<>(elements);
 	}
 
+	/**
+	 * Gets the unserialized contents of this {@code Tuple} without copying into a new list.
+	 *
+	 * @return the elements that make up this {@code Tuple}.
+	 */
+	public List<Object> getRawItems() {
```

Comment:
How about package protected to keep to the spirit of hiding the raw list?

Reply:
Like it!!!

Reply:
Sounds good.